app/models/miq_product_features_share.rb

```diff
@@ -0,0 +1,2 @@
+class MiqProductFeaturesShare < ApplicationRecord
```

Comment:
Is this the correct filename and model name?  Note the plural / singular and the order.